ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/PrivacyApiGroupJsonRpcMethods.java

```diff
@@ -76,9 +78,14 @@ public PrivacyParameters getPrivacyParameters() {
         createPrivateMarkerTransactionFactory(
             privacyParameters, blockchainQueries, transactionPool.getPendingTransactions());
 
-    final PrivacyController privacyController =
-        new PrivacyController(
+    final DefaultPrivacyController defaultPrivacyController =
```

Comment:
How about moving the creation into a new private method:
```
   final PrivacyController privacyController = privacyController()
    ...
}

private PrivacyController privacyController(){
    return privacyParameters.isMultiTenancyEnabled()
             ? new MultiTenancyPrivacyController(
                 defaultPrivacyController, privacyParameters.getEnclave())
             : new DefaultPrivacyController(
             privacyParameters, protocolSchedule.getChainId(), markerTransactionFactory);;
}
```

Besides making the create methods short & nicer to read, as a bonus It'll only create the default controller when it is needed.

Reply:
good idea